core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -172,6 +173,15 @@ public static boolean isPersistNotify() {
     }
 
     /**
+     * Returns true if notifications are persistent
+     *
+     * @return {@code true} if notifications are persistent, {@code false}  otherwise
+     */
+    public static boolean setLockscreenBackground() {
+        return prefs.getBoolean(PREF_LOCKSCREEN_BACKGROUND, false);
```

Comment:
I think 'false' needs to get changed to true. For some reason the 'default' in the preferences doesn't work otherwise. It shows up as being checked but doesn't actually work until you uncheck/check it again.  I'm not sure why it works this way. :(  (Tested on 5.0.0 and 5.1)